### DISCUSSION TITLE
'An error occured'-message in my log drains

### DISCUSSION THREAD

**Discussion by bergiusj** [vercel/community#256] [Help] [answered]
Anyone other than me that's been having issues with their log drains the last 24 hours? I think Vercel had a problem a day ago with the builds and logs, but it should've been fixed according to the status page. I use two different log drains and they both report the same thing, so the problem should not be from the external log drains

`static 200 GET /_next/image?url=%2F_next%2Fstatic%2Fimage%2Fpublic%2Fcoworking_illustration.8833012ee5b9d22855a8451d4665cce7.png&w=384&q=75 3.68.92.226 : An error occurred
` 
So I get 200 OK, but at the end it says: 'An error occured'.

Any clues?

**Comment by freddy38510** [comment]
The same issue occurs for me since yesterday.

In the `message` field the value is exactly `: An error occurred`.

This happens for static sources and lambda sources when the `proxy.cache_id` field has a value.

```json
{
  "dt":"2022-01-26 15:13:01.615 UTC",
  "message_string":": An error occurred",
  "platform_string":"Vercel",
  "vercel":  {
    "deployment_id_string":"****",
    "host_string":"****",
    "id_string":"1643209981518967905770802255",
    "path_string":"/",
    "project_id_string":"****",
    "proxy": {
      "cache_id_string":"29h28-1643126656250-aba81e54da86",
      "client_ip_string":"x.x.x.x",
      "errorCode_string":"",
      "host_string":"bal.wapesleague.com",
      "method_string":"GET","path_string":"/api/bal?position=GK&style=phy&height=180",
      "referer_string":"https://bal.wapesleague.com/",
      "region_string":"cdg1",
      "scheme_string":"https",
      "status_code_integer":"200",
      "timestamp_integer":"1643209981516",
      "user_agent_string":"[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36\"]"
    },
  "request_id_string":"kcbtl-1643209981516-7e2a81294250",
  "source_string":"lambda",
  "source_type_string":"http"
  }
}
```

**Comment by JTaylor0196** [comment]
Hey all, thanks for your input! As a result of this feedback, the team is looking into this and we'll let you know what we find. Stay tuned!

**Comment by JTaylor0196** [comment, marked as answer]
Hey there @bergiusj & @freddy38510! Circling back to update you all and let you know that the team created and rolled out a fix for this issue today. Please let me know if you still happen to experience this issue, but, moving forward, you should be all set.

Thanks for your patience while we've worked through this!
